src/custom/components/AccountDetails/Transaction/index.tsx

```diff
@@ -137,17 +137,19 @@ function ActivitySummary(params: {
       )
     }
 
+    const getPriceFormat = ({ price, kind }: { price: string; kind: string }): string => {
+      return `${price} ${kind === 'buy' ? outputAmount.currency.symbol : sellAmt.currency.symbol} per ${
+        kind === 'buy' ? sellAmt.currency.symbol : outputAmount.currency.symbol
+      }`
+    }
```

Comment:
should this be in a more central place? maybe can be reused for other parts

Reply:
The only place I can imagine is the price we use on the swap container/confirmation modal. Not sure how much of that code overlaps. You think that should be addressed in this PR or can it be a future iteration?

Reply:
Reason I created a separate function here, is because otherwise there would be repetitive similar (not dry) code between the limitPrice and executionPrice.

Reply:
I don't think this is right.
See my comment with the examples.

Only makes sense to have this logic if we are inverting the prices.